Docs/source/running_cpp/parameters.rst

```diff
@@ -44,6 +44,17 @@ Overall simulation parameters
 * ``warpx.verbose`` (`0` or `1`)
     Controls how much information is printed to the terminal, when running WarpX.
 
+* ``warpx.random_seed`` (`string` or `int` >= 1) optional
+    If provided ``warpx.random_seed = random``, the random seed will be determined
+    using `std::random_device` and `std::clock()`,
+    thus every simulation run produces different random numbers.
+    If provided ``warpx.random_seed = n``, and it is required that `n >= 1`,
+    the random seed for each MPI rank is `(mpi_rank+1) * n`,
+    where `mpi_rank` starts from 0.
+    Note that `n = 1` produces the default random seed.
```

Comment:
To be complete, you can mention the string 'default' as a valid value, equivalent to setting n = 1.